modules/test/playwright/pages/account-admin-web/EditAccountPage.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+// @ts-ignore
+
+import {Locator, Page} from '@playwright/test';
+
+export class EditAccountPage {
```

Comment:
Can we separate the contact stuff on a dedicated page? Like EditAccountContactPage... otherwise all the things related to contact, addresses, details will be here and it will became very complicated to maintain

Reply:
Same for the addresses. I would like to have a dedicated page to manage them

Reply:
Moved contact related stuff to EditAccountContactPage, removed addressLink for now as it is not being used in this PR. We can add it back/create a new page with the other ticket.